lib/presentation/extensions/send_file_extension.dart

```diff
@@ -329,4 +335,11 @@ extension SendFileExtension on Room {
       return null;
     }
   }
+
+  Future<String> _generateBlurHash(String filePath) async {
+    final data = File(filePath).readAsBytesSync();
+    final image = img.decodeImage(data);
+    final blurHash = BlurHash.encode(image!, numCompX: 4, numCompY: 3);
```

Comment:
why forcing image nullity ? are you sure image can't be null ?

Reply:
fxied